### PR TITLE
Change numpy2floattensor funciton

### PR DESCRIPTION
**Pull request by jiseongHAN**
#243 #261 
Change numpy2floattensor function to prevent memory leak.

All torch.FloatTensor  changes to numpy2floattensor.

**lunarlander_continuous environment**
a2c ddpg TD3SACppobcddpgbcsacddpgfdsacfd

A2C / DDPG / TD3 / SAC / PPO / BC_DDPG / BC_SAC / DDPGfD / SACfD

**lunarlander environment**
dqn_lunardqnfd_lunar!r2d1_lunardistllation_lunar

DQN / DQNfD / R2D1 / Distillation DQN

**pong_no_frameskip environment**
dqn_pongapex_pongr2d1_pongdqn_resnet_pongdistill_pong

DQN / Apex / R2D1 / DQN with Resnet / Distillation

### CODE REVIEW

File: configs/lunarlander_continuous_v2/a2c.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - Author: Kyunghwan Kim
 - Contact: kh.kim@medipixel.io
 """
+
```
Comment (MrSyee) [inline configs/lunarlander_continuous_v2/a2c.py:6]: useless a new line

File: rl_algorithms/common/helper_functions.py [hunk 2]
```diff
@@ -91,13 +91,28 @@ def get_n_step_info(
...
 
-def numpy2floattensor(arrays: Tuple[np.ndarray]) -> Tuple[np.ndarray]:
-    """Convert numpy arrays to torch float tensor."""
-    tensors = []
-    for array in arrays:
-        tensor = torch.FloatTensor(array).to(device, non_blocking=True)
-        tensors.append(tensor)
-    return tuple(tensors)
+def numpy2floattensor(arrays: Tuple[np.ndarray], device_: str) -> Tuple[torch.Tensor]:
+    """
```
Comment (MrSyee) [inline rl_algorithms/common/helper_functions.py:95]: should modify docstring

File: rl_algorithms/distillation/dqn_agent.py [hunk 3]
```diff
@@ -218,7 +216,7 @@ def add_expert_q(self):
             with open(file_name_list[i], "rb") as f:
                 state = pickle.load(f)[0]
 
-            torch_state = torch.from_numpy(state).float().to(device)
+            torch_state = torch.from_numpy(state).float().to(self.device)
```
Comment (MrSyee) [inline rl_algorithms/distillation/dqn_agent.py:219]: should use `numpy2floattensor` method

File: rl_algorithms/dqn/linear.py [hunk 4]
```diff
@@ -62,7 +60,11 @@ def reset_parameters(self):
     @staticmethod
     def scale_noise(size: int) -> torch.Tensor:
         """Set scale to make noise (factorized gaussian noise)."""
-        x = torch.FloatTensor(np.random.normal(loc=0.0, scale=1.0, size=size))
+        x = (
```
Comment (MrSyee) [inline rl_algorithms/dqn/linear.py:63]: should use `numpy2floattensor` method

File: tools/run_pong.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+python run_pong_no_frameskip_v4.py --cfg-path ./configs/pong_no_frameskip_v4/apex_dqn.py --off-render --log
```
Comment (MrSyee) [inline tools/run_pong.sh:1]: should remove this file

**Review by jinPrelude — APPROVED** [review]

**Comment by MrSyee** [comment]
@all-contributors please add @jiseongHAN for code

**Review by MrSyee — APPROVED** [review]
Good job!

**Pull request merged**